metadata.rb

```diff
@@ -7,7 +7,7 @@
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 source_url      'https://github.com/rackspace-cookbooks/ohai_public_info'
 issues_url       'https://github.com/rackspace-cookbooks/ohai_public_info/issues'
-version          '0.1.0'
+version          '0.1.1'
 
 depends 'ohai'
 depends 'poise'
```

Comment:
This should probably pin to `~> 2.0.0` so Berkshelf doesn't give a downstream cookbook v1 of poise with this cookbook.